tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -1691,6 +1692,26 @@ public function theDataOfTheResponseShouldMatch(
 		);
 	}
 
+	/**
+	 * @Then the JSON data of the response should match
+	 *
+	 * @param PyStringNode|string $schemaString
+	 * @param object|null $response
+	 *
+	 * @return void
+	 *
+	 * @throws Exception
+	 */
+	public function theDataOfTheResponseShouldMatch($schemaString, ?object $response = null): void {
+		if ($response === null) {
+			$response = $this->getJsonDecodedResponseBodyContent();
+		}
+		JsonAssertions::assertJsonDocumentMatchesSchema(
+			$response,
+			$this->getJSONSchema($schemaString)
+		);
+	}
```

Comment:
we usually refer `$response` as a while response object (i.e. Guzzle response)
```suggestion
	public function theDataOfTheResponseShouldMatch($schemaString, ?object $responseBody = null): void {
		if ($responseBody === null) {
			$responseBody = $this->getJsonDecodedResponseBodyContent();
		}
		JsonAssertions::assertJsonDocumentMatchesSchema(
			$responseBody,
			$this->getJSONSchema($schemaString)
		);
	}
```